psealock/gutenberg
# Use css-in-js for styling in navigation

## Description
Migrates styles from scss to css-in-js using emotion.

## Screenshots 
<img width="331" alt="Screen Shot 2020-08-11 at 5 35 59 PM" src="https://user-images.githubusercontent.com/10561050/89951292-d3053c80-dc33-11ea-99de-8df290544c6d.png">



## Testing

1. Run `npm run storybook:dev`.
1. Make sure that the `Navigation` component renders appropriately matching the design.
1. Make sure when clicking the buttons, the full width of the button inside the menu is focused.
